src/test/resources/pages/instructorFeedbackResultsFilteredByNoSection.html

```diff
@@ -560,7 +560,7 @@
         </span>
       </div>
     </div>
-    <div class="panel-collapse collapse in" id="panelBodyCollapse-7" style="height: auto;">
+    <div class="panel-collapse collapsing" id="panelBodyCollapse-7" style="height: 50px;">
```

Comment:
This shouldn't be part of the change. Anyway, occurrences of this was fixed in the latest `master` branch so you need to sync, and undo changes like this.